CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 * Add auto-completion support for step parameters with limited number of choices
+* Enable comments toggling in story and steps files
```

Comment:
we need to document, how to comment/uncomment line

Reply:
not sure how to do this on windows :(

Reply:
added